### PR TITLE
imessage-exporter 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by unitof**
Revival of #124020

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by unitof** [comment]
Description does not start with capital letter following typical style for `iMessage` brand.

File: Formula/imessage-exporter.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class ImessageExporter < Formula
+  desc "iMessage conversation database export and inspector"
```
[the remark this answers is not in the document]
Comment (unitof) [reply Formula/imessage-exporter.rb]: Rewrote to `"Command-line tool to export and inspect local iMessage database"` just before you left this review, but happy to change.

[the remark this answers is not in the document]
Comment (dtrodrigues) [reply Formula/imessage-exporter.rb]: Your new wording is fine, I was just offering a suggestion to prevent the leading `i`.

**Review by dtrodrigues — APPROVED** [review]

**Comment by dtrodrigues** [comment]
Thanks @unitof

**Pull request merged**